tests/dummy/app/styles/app.scss

```diff
@@ -40,29 +39,29 @@ h3 {
   background-image: linear-gradient(-134deg, #3023AE 0%, #A56DD7 100%);
   -webkit-display: flex;
   display: flex;
-  @include justify-content(center);
-  @include align-items(center);
-  @include flex-direction(column);
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
```

Comment:
#178 adds postcss and autoprefixer to the dummy app to accommodate for these changes.